packages/components/src/components/Snackbar/Snackbar.tsx

```diff
@@ -155,23 +127,16 @@ export const Snackbar: FC<SnackbarProps> = (toast) => {
     _20: 20,
     _24: 24,
   }
```

Comment:
Can't comment on the appropriate lines since they weren't edited, but lines 89-105 should be updated to document the updated explanation for how to implement the Snackbar within an app.

Reply:
Updated the docs